spec/system/candidate_interface/carry_over/candidate_carries_over_different_application_states_spec.rb

```diff
@@ -87,6 +122,18 @@ def then_i_can_carry_over_my_application_to_the_new_cycle
     and_i_should_be_in_your_details_page
   end
 
+  def then_i_can_carry_over_my_application_to_the_new_cycle_using_the_carry_over_interstitial
+    and_i_am_signed_in_as_a_candidate
```

Comment:
And this as well.

Reply:
Same as above, I removed the second time sign in